FWCore/ParameterSet/python/Config.py

```diff
@@ -151,6 +151,9 @@ def filterNames(self):
     def pathNames(self):
         """Returns a string containing all the Path names separated by a blank"""
         return ' '.join(self.paths_().keys())
+    def subProcessNames(self):
+        """Returns a string containing all the subProcess names separated by a blank"""
+        return ' '.join(self.subProcesses_().keys())
```

Comment:
You're right. It was left over from the old implementation. Since this function is not used anywhere, I removed it in the updated PR.